PyISY/Nodes/group.py

```diff
@@ -21,11 +23,12 @@ class Group(object):
     status = Property(0)
     hasChildren = False
 
-    def __init__(self, parent, nid, name, members=[]):
+    def __init__(self, parent, nid, name, members=[], controllers=[]):
```

Comment:
I'm a bit surprised I didn't catch this before:

Both members and controllers should not have a list as their default value. It should be None instead.